angular/src/js/products.js

```diff
@@ -0,0 +1,104 @@
+(function () {
+    var app = angular.module("BetterThanEtsy", []);
+
+    //    app.controller('StoreController', ['$http', function($http){
+    ////       this.items = products;
+    //         var self = this;
+    //         self.items = [];
+    //        
+    //    
+    //        
+    //       $http.get('js/etsy.json').succes(function(data) {
+    //           self.items = data;
+    //        
+    //    })
+    //    } ])
```

Comment:
It's working code...It show us how data looks ugly on our HTML  if nothing is done to render it properly.  But, yeah... It's being deleted and saved somewhere else. :)